fem/estimators.cpp

```diff
@@ -329,7 +329,7 @@ void KellyErrorEstimator::ComputeEstimates()
          error_estimates(e) = sqrt(factor * error_estimates(e));
       }
 
-      total_error = error_estimates.Sum();
+      total_error = error_estimates.Norml2();
```

Comment:
Nice catch, but a better fix seems to be removing the `sqrt` for each `e` and keeping the `Sum`, then taking `sqrt` of the final result. This may be slightly faster (fewer `sqrt` calls) and more numerically accurate.

Reply:
You don't want to do that: the array `error_estimates` is meant to approximate the norm of the local error, not its square.

Reply:
Good point, I forgot `error_estimates` is used later (just looking at the `total_error` computation) when I made this comment.